### ISSUE TITLE
Way to view entire timeline item description

### ISSUE THREAD

**Issue by connor4312** [microsoft/vscode#91704]
Refs: https://github.com/microsoft/vscode/issues/91330

I tried to insert a big lorem ipsum paragraph. It was truncated correctly but it doesn't look like there's a way to view the whole thing, maybe it should be alt text?

![image](https://user-images.githubusercontent.com/2230985/75468106-8eb20880-5941-11ea-9db0-b5e1d0232cad.png)

**Comment by eamodio** [comment]
That's up to the provider to provide the full description in the details (which is shown on hover). Which is the same as our trees
